barretenberg/cpp/src/barretenberg/eccvm_recursion/ecc_lookup_relation.cpp

```diff
@@ -0,0 +1,11 @@
+#include "barretenberg/eccvm_recursion/eccvm_recursive_flavor.hpp"
+#include "barretenberg/flavor/relation_definitions.hpp"
+#include "barretenberg/relations/ecc_vm/ecc_lookup_relation_impl.hpp"
+#include "barretenberg/stdlib/primitives/bigfield/bigfield.hpp"
+
+namespace bb {
```

Comment:
the linking magic that we have also done for the translator relations to be able to instantiate them on bigfield but only where needed